.gitignore

```diff
@@ -1,3 +1,8 @@
+# General files
+*~
+.DS_Store
+
+# Build files
```

Comment:
I don't think there is enough in this file to warrant these headings. Just add `*~` and `.DS_Store` to the file.

Reply:
Agree 👍